test_data/BCP00301/README.md

```diff
@@ -67,7 +67,7 @@ require that the unit under test communicate directly with entities which are 'm
 Additional certificates can be generated using this CA if required, using the './generateCerts' script followed by a
 domain name and an optional additional domain name ending in '.local' for mDNS usage.
```

Comment:
[...]followed by a _hostname,_ domain name and an optional additional _fully-qualified_ domain name ending in[...]